setup.cfg

```diff
@@ -34,3 +34,10 @@ test_suite = ginga.tests.ginga_test_suite
 [entry_points]
 ginga = ginga.rv.main:_main
 ggrc = ginga.misc.grc:_main
+
+[flake8]
+# Ignoring these for now:
+# E265: block comment should start with '#'
+# E266: too many leading '#' for block comment
+# E501: line too long
+ignore = E265,E266,E501
```

Comment:
Note: This is mostly for my own sanity. What this does it that if you use `flake8` as your PEP 8 checker, it will find this file and ignore the given error/warning codes here.